### PR TITLE
Untrademark json files

### PR DESCRIPTION
**Pull request by zuiderkwast**
Part of #25

Covers `src/commands/*.json` and `src/commands/README.md`

### CODE REVIEW

**Comment by zuiderkwast** [comment]
This one just removes the word Redis or replaces it with server. It's independent of the name we pick for the project. It's strings visible in documentation, so we want to change this. Thus, ready to review imo.

File: src/commands/command-docs.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
                             "type": "string"
                         },
                         "since": {
-                            "description": "the Redis version that added the command (or for module commands, the module version).",
+                            "description": "the server verion that added the command (or for module commands, the module version).",
```
Comment (madolson) [inline src/commands/command-docs.json:35]:
```suggestion
                            "description": "the server version that added the command (or for module commands, the module version).",
```

File: src/commands/command-docs.json [hunk 2]
```diff
@@ -118,7 +118,7 @@
                             }
                         },
                         "deprecated_since": {
-                            "description": "the Redis version that deprecated the command (or for module commands, the module version)",
+                            "description": "the server verion that deprecated the command (or for module commands, the module version)",
```
Comment (madolson) [inline src/commands/command-docs.json:121]:
```suggestion
                            "description": "the server version that deprecated the command (or for module commands, the module version)",
```

File: src/commands/command-docs.json [hunk 3]
```diff
@@ -135,7 +135,7 @@
                                 "items": [
                                     {
                                         "type": "string",
-                                        "description": "The Redis version that the entry applies to."
+                                        "description": "The server verion that the entry applies to."
```
Comment (madolson) [inline src/commands/command-docs.json:138]:
```suggestion
                                        "description": "The server version that the entry applies to."
```

File: src/commands/lolwut.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
     "LOLWUT": {
-        "summary": "Displays computer art and the Redis version",
+        "summary": "Displays computer art and the server verion",
```
Comment (madolson) [inline src/commands/lolwut.json:3]:
```suggestion
        "summary": "Displays computer art and the server version",
```

File: src/commands/lolwut.json [hunk 5]
```diff
@@ -11,7 +11,7 @@
         ],
         "reply_schema": {
             "type": "string",
-            "description": "String containing the generative computer art, and a text with the Redis version."
+            "description": "String containing the generative computer art, and a text with the server verion."
```
Comment (madolson) [inline src/commands/lolwut.json:14]:
```suggestion
            "description": "String containing the generative computer art, and a text with the server version."
```

**Review by madolson — COMMENTED** [review]
I think I like all of this agnostic descriptions more. Including Redis in the Redis docs did always seem verbose.

File: src/commands/command-docs.json (hunk 1 above)
Comment (zuiderkwast) [reply src/commands/command-docs.json:35]: Yeah I misspelled in the mass replace. :sleepy:

**Comment by zuiderkwast** [comment]
I accidentally created this branch on placeholderkv repo instead of on my own dev fork. I'll need to reopen it as another PR.

**Comment by zuiderkwast** [comment]
So currently I can't even delete the branch on this repo, i.e. https://github.com/placeholderkv/placeholderkv/tree/untrademark-json-files

Btw I think we clean up old branches, remains from many years ago.

**Pull request closed without merging**